rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

```diff
@@ -0,0 +1,144 @@
+= Managing Automations
+
+When a project manager clicks the icon *Start automation* in the *My Backlog* view of the *My RPA* module, he creates a process automation project assigned to a category in the *Process Management* view of the *Process Automation* module.
```

Comment:
he creates > RPA Manager creates... 
I'm suggesting this just to avoid using "he" to refer to a project manager. Please check other instances where this is mentioned.